src/lib.rs

```diff
@@ -2340,7 +2340,40 @@ impl Connection {
             }
         }
 
-        // Create PING for PTO probe.
+        // Try to retransmit old frames on PTO, if there is space left.
+        if self.recovery.loss_probes[epoch] > 0 && left >= 1 && !is_closing {
```

Comment:
is `left >= 1` the right thing to do here? Maybe take into account CRYPTO ¦ STREAM frame overheads?

Reply:
I think it's a copy of L2377 and just to fail early when there is no space to add a frame.

Reply:
We can probably check for min(CRYPTO_OVERHEAD, STREAM_OVERHEAD), or something like that.

Reply:
Done.